src/test/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPluginSpec.groovy

```diff
@@ -44,6 +56,7 @@ final class CheckerFrameworkPluginSpec extends BaseSpecification {
       .withGradleVersion(gradleVersion)
       .withProjectDir(testProjectDir.root)
       .withPluginClasspath()
+      .withArguments('--debug', '--stacktrace')
```

Comment:
Is `--debug` really needed? AFAIK `--debug` produces extreme amount of logging in `$HOME/.gradle/daemon/*` which is never removed by Gradle :-/

Reply:
Removed — accidentally left it in while testing.